ITEventsUITests/FullEventViewUITests.swift

```diff
@@ -0,0 +1,114 @@
+import XCTest
+
+class FullEventViewUITests: XCTestCase {
+    private var app: XCUIApplication!
+    
+    private lazy var eventView = app.otherElements.matching(identifier: "EventView")
+    private lazy var eventImage = app.images.matching(identifier: "EventImage")
+    private lazy var eventTitle = app.staticTexts.matching(identifier: "EventTitle")
+    private lazy var eventDate = app.staticTexts.matching(identifier: "EventDate")
+    private lazy var eventCity = app.staticTexts.matching(identifier: "EventCity")
+    private lazy var eventAddress = app.staticTexts.matching(identifier: "EventAddress")
+    private lazy var eventDescription = app.staticTexts.matching(identifier: "EventDescription")
+    private lazy var eventTags = app.otherElements.matching(identifier: "EventTags")
+    private lazy var eventSourceButton = app.buttons.matching(identifier: "EventSourceButton")
+    private lazy var showSimilarEventsButton = app.otherElements["ShowSimilarEventsButton"]
+        
+    override func setUp() {
+        super.setUp()
+        continueAfterFailure = false
+        
+        app = XCUIApplication()
+        app.launch()
+    }
+    
+    override func tearDown() {
+        super.tearDown()
+    }
+    
+    func testFullEventViewContent() {
+        app.collectionViews.cells.element(boundBy: 0).tap()
```

Comment:
Can we find the right collectionView, instead of working with all possible collectionViews (applied to all tests here):
```swift
private lazy var requiredCollectionView = app.collectionViews.matching(identifier: "RequiredCollectionView")

...
requiredCollectionView.cells.element(boundBy: 0).tap()
```

Don't you want to have here the blocks separated by the following comments: Given, When, Then ?